zenodo/modules/frontpage/templates/zenodo_frontpage/index.html

```diff
@@ -44,7 +44,7 @@
         {%- set publication_date = r.publication_date|from_isodate -%}
         {%- set creation_date = r._created|from_isodatetime -%}
         {%- set embargo_date = r.embargo_date|from_isodate -%}
-        {%- set object_type = r.resource_type|objecttype %}
+        {%- set object_type = r.resource_type.to_dict()|objecttype %}
```

Comment:
I agree that the fix is for es dsl - can you open an issue, but in the mean time move the logic to ``ObjectType`` model (e.g. the ``to_dict()`` i don't think will work if the record suddenly comes from the db).